docs/book/platform-guide/set-up-your-mlops-platform/deploy-zenml/deploy-zenml.md

```diff
@@ -11,14 +11,13 @@ A ZenML deployment typically looks like this:
+  * the individual orchestrators and step operators communicate with the server to write and track the pipeline run data
+  * the dashboard is served from the server to give a UI interface to all the metadata
 * An **SQL database** that acts as the backend to track configurations and metadata
-    * for production, currently, only MySQL is supported
+  * for production, currently, only MySQL is supported
 * An optional **secrets management service** that is used as a backend for the ZenML secrets store
 
 Choose the most appropriate deployment strategy for you out of the following options to get started with the deployment:
 
-<table data-card-size="large" data-view="cards"><thead><tr><th></th><th></th><th data-hidden></th><th data-hidden data-type="content-ref"></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td><mark style="color:purple;"><strong>Deploy with ZenML CLI</strong></mark></td><td>Deploying ZenML on cloud using the ZenML CLI.</td><td></td><td></td><td><a href="broken-reference">Broken link</a></td></tr><tr><td><mark style="color:purple;"><strong>Deploy with Docker</strong></mark></td><td>Deploying ZenML in a Docker container.</td><td></td><td></td><td><a href="deploy-with-docker.md">deploy-with-docker.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Deploy with Helm</strong></mark></td><td>Deploying ZenML in a Kubernetes cluster with Helm.</td><td></td><td></td><td><a href="deploy-with-helm.md">deploy-with-helm.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Deploy using HuggingFace Spaces</strong></mark></td><td>Deploying ZenML to Huggingface Spaces.</td><td></td><td></td><td><a href="deploy-using-huggingface-spaces.md">deploy-using-huggingface-spaces.md</a></td></tr></tbody></table>
+<table data-card-size="large" data-view="cards"><thead><tr><th></th><th></th><th data-hidden></th><th data-hidden data-type="content-ref"></th><th data-hidden data-card-target data-type="content-ref"></th></tr></thead><tbody><tr><td><mark style="color:purple;"><strong>Deploy with ZenML CLI</strong></mark></td><td>Deploying ZenML on cloud using the ZenML CLI.</td><td></td><td></td><td><a href="deploy-with-zenml-cli.md">deploy-with-zenml-cli.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Deploy with Docker</strong></mark></td><td>Deploying ZenML in a Docker container.</td><td></td><td></td><td><a href="deploy-with-docker.md">deploy-with-docker.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Deploy with Helm</strong></mark></td><td>Deploying ZenML in a Kubernetes cluster with Helm.</td><td></td><td></td><td><a href="deploy-with-helm.md">deploy-with-helm.md</a></td></tr><tr><td><mark style="color:purple;"><strong>Deploy using HuggingFace Spaces</strong></mark></td><td>Deploying ZenML to Huggingface Spaces.</td><td></td><td></td><td><a href="deploy-using-huggingface-spaces.md">deploy-using-huggingface-spaces.md</a></td></tr></tbody></table>
```

Comment:
```suggestion
<table data-card-size="large" data-view="cards">
    <thead>
        <tr>
            <th></th>
            <th></th>
            <th data-hidden></th>
            <th data-hidden data-type="content-ref"></th>
            <th data-hidden data-card-target data-type="content-ref"></th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>
                <mark style="color:purple;"><strong>Deploy with ZenML CLI</strong></mark>
            </td>
            <td>Deploying ZenML on cloud using the ZenML CLI.</td>
            <td></td>
            <td></td>
            <td>
                <a href="deploy-with-zenml-cli.md">deploy-with-zenml-cli.md</a>
            </td>
        </tr>
        <tr>
            <td>
                <mark style="color:purple;"><strong>Deploy with Docker</strong></mark>
            </td>
            <td>Deploying ZenML in a Docker container.</td>
            <td></td>
            <td></td>
            <td>
                <a href="deploy-with-docker.md">deploy-with-docker.md</a>
            </td>
        </tr>
        <tr>
            <td>
                <mark style="color:purple;"><strong>Deploy with Helm</strong></mark>
            </td>
            <td>Deploying ZenML in a Kubernetes cluster with Helm.</td>
            <td></td>
            <td></td>
            <td>
                <a href="deploy-with-helm.md">deploy-with-helm.md</a>
            </td>
        </tr>
        <tr>
            <td>
                <mark style="color:purple;"><strong>Deploy using HuggingFace Spaces</strong></mark>
            </td>
            <td>Deploying ZenML to Huggingface Spaces.</td>
            <td></td>
            <td></td>
            <td>
                <a href="deploy-using-huggingface-spaces.md">deploy-using-huggingface-spaces.md</a>
            </td>
        </tr>
    </tbody>
</table>
```

Even doing something like this would already make a huge difference...